### PR TITLE
Add extra steps to new machine setup

### PR DESCRIPTION
**Pull request by sedders123**
These extra tweaks were needed when setting up a compbox for the upcoming friendly sessions

### CODE REVIEW

File: new-machine.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 3. Create a non-root user with `sudo` access:
 
     ```bash
-    useradd --create-home --user-group --groups sudo $USERNAME
+    useradd --create-home --user-group --groups sudo $USERNAME --s /bin/bash
```
Comment (PeterJCLaw) [inline new-machine.md]: Should this be `--shell`? I'm assuming this is a typo?
```suggestion
    useradd --create-home --user-group --groups sudo $USERNAME --shell /bin/bash
```

File: new-machine.md [hunk 2]
```diff
@@ -43,13 +43,17 @@
 
 9. Set up public DNS for the machine.
 
-10. Run the install:
+10. Edit `/etc/puppet/modules/compbox/manifests/init.pp` to change the `$compstate` variable to point at your compstate.
```
Comment (PeterJCLaw) [inline new-machine.md]: This should not be needed. It is expected and intended that the default deploy is the dummy compstate. You should then just deploy your actual one over the top (hence the comment about being aware of the warnings on the first deploy [in the readme](https://github.com/PeterJCLaw/srcomp-puppet#deployment)).

**Comment by PeterJCLaw** [comment]
Thanks for putting this together. One small thing (I'm assuming a typo) and one correction.

File: new-machine.md (hunk 2 above)
Comment (sedders123) [reply new-machine.md]: Ahh apologies, missed that when skimming the README, shall read in more detail next time!

Comment (sedders123) [reply new-machine.md]: 0e2dc43

Comment (PeterJCLaw) [reply new-machine.md]: Hrm, I wonder if the last step in the machine setup should be something like "now deploy your compstate using `srcomp deploy` locally"?

Comment (sedders123) [reply new-machine.md]: That sounds like a good idea!

Comment (sedders123) [reply new-machine.md]: 051257e

**Comment by PeterJCLaw** [comment]
Thanks!

**Pull request merged**